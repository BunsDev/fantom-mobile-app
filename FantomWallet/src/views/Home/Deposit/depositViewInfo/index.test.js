import React from 'react';
import { shallow } from 'enzyme';

// eslint-disable-next-line import/first
import { DepositViewInfo } from './index';

test('renders correctly', () => {
  const wrapper = shallow(<DepositViewInfo publicKey="0x00023" />);
  expect(wrapper).toMatchSnapshot();
});
