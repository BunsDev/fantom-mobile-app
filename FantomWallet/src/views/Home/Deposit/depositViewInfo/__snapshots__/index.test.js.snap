// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ScrollViewMock
  showsVerticalScrollIndicator={false}
  style={
    Object {
      "height": 1334,
    }
  }
>
  <View
    style={
      Object {
        "marginBottom": 26.68,
        "marginTop": 12,
        "padding": 16,
      }
    }
  >
    <QRGenerator
      billingAmount=""
      qrLink=""
      titleText="Address QR Code"
    />
    <ForwardRef(QRCodeSave)
      amount=""
      content=""
    />
  </View>
  <BillingAmountScreen
    headerText="FTM"
    onAmountChange={[Function]}
    onTextFieldBlur={[Function]}
    onTextFieldFocus={[Function]}
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "marginTop": 40.019999999999996,
      }
    }
  >
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "rgb(29,33,38)",
          "borderRadius": 75,
          "height": 150,
          "justifyContent": "center",
          "marginBottom": 10,
          "width": 150,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgb(44,52,58)",
            "borderRadius": 60,
            "height": 120,
            "justifyContent": "center",
            "width": 120,
          }
        }
      >
        <Icon
          allowFontScaling={false}
          color="#FFF"
          name="share-apple"
          size={67.5}
        />
      </View>
    </TouchableOpacity>
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontFamily": "SFProDisplay-Semibold",
          "fontSize": 16,
        }
      }
    >
      Share
    </Text>
  </View>
  <View
    style={
      Object {
        "height": 200.1,
        "marginBottom": 10,
      }
    }
  />
</ScrollViewMock>
`;
